packages/core/src/components/MonthPicker/MonthPicker.jsx

```diff
@@ -76,7 +78,19 @@ export class MonthPicker extends React.PureComponent {
       } else {
         selectedMonths.push(month);
       }
+
       this.setState({ selectedMonths });
+
+      this.handleAriaPressed(selectedMonths);
+    }
+  }
+
+  handleAriaPressed(selectedMonths) {
+    if (!this.isControlled) {
+      this.setState({
+        selectAllPressed: selectedMonths.length === 12,
```

Comment:
What happens when some months are disabled? For example, if `4` months are disabled, should `selectAllPressed` be `true` when `8` are selected? Instead of hard-coding `12` here and elsewhere, should this instead reference the number of active checkboxes?